ticketutil/servicenow.py

```diff
@@ -2,18 +2,11 @@
 import logging
 import requests
 
-from . import ticket
+# from . import ticket
```

Comment:
I copied this line from jira.py, but during unit tests I realized that's not going to work. I can remove it I guess.